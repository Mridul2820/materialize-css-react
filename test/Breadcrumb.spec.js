import React from 'react';
import { render } from '@testing-library/react';
import Breadcrumb from '../src/Breadcrumb';

describe('<Breadcrumb />', () => {
  test('renders', () => {
    const { container } = render(
      <Breadcrumb cols={6} className="blue">
        <a href="#">Home</a>
        <a href="#">Login</a>
      </Breadcrumb>
    );

    expect(container).toMatchSnapshot();
    expect(container.querySelectorAll('.breadcrumb').length).toBe(2);
    expect(container.querySelector('.row.blue')).toBeTruthy();
  });
});
