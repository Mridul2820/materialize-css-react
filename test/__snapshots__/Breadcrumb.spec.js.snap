// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> renders 1`] = `
<div>
  <nav
    class="row blue"
  >
    <div
      class="nav-wrapper"
    >
      <div
        class="col s6"
      >
        <a
          class="breadcrumb"
          href="#"
        >
          Home
        </a>
        <a
          class="breadcrumb"
          href="#"
        >
          Login
        </a>
      </div>
    </div>
  </nav>
</div>
`;
